### PR TITLE
Fix media browser panel title + selection header color

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

Adds proper translation key for the media browser panel. This is related to the core PR https://github.com/home-assistant/core/pull/39720 that ensures the proper key is used for the panel title.

This PR also fixes the wrong color of the heading of the "Choose Player" dialog (was using black also in dark mode).

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-dialog.ts [hunk 1]
```diff
@@ -78,6 +78,7 @@ export class HaDialog extends MwcDialog {
           color: inherit;
         }
         .header_title {
+          color: var(--primary-text-color);
```
Comment (bramkragten) [inline src/components/ha-dialog.ts:81]: Why is this needed now and not before?
```suggestion
          color: var(--mdc-dialog-heading-ink-color);
```

Comment (spacegaier) [reply src/components/ha-dialog.ts:81]: Your requested change does not work. It would once again result in a black header color since `--mdc-dialog-heading-ink-color` is not defined, which is weird since it is set here: https://github.com/home-assistant/frontend/blob/349a5f52b10691416962ac707deaf5cf3ce030af/src/resources/styles.ts#L291

It for example gets set directly for one other media player related dialog (in `hui-dialog-web-browser-play-media.ts`), but I think it would be better to set it once for all dialogs in `.mdc-dialog`.

No idea why no one came across this so far. I myself only recently switched to the dark default theme.

Comment (bramkragten) [reply src/components/ha-dialog.ts:81]: This was working before, as like you linked `mdc-dialog-heading-ink-color` is defined and should be used, so something changed that broke it. We should find that.\

The overruling in `hui-dialog-web-browser-play-media` should be removed.

Comment (spacegaier) [reply src/components/ha-dialog.ts:81]: I think the problem could come from the fact this hides the previous `ha-dialog` style:
https://github.com/home-assistant/frontend/blob/e63a78bcdb556dcb1e64605ec4145ca7e956e12a/src/panels/media-browser/hui-dialog-select-media-player.ts#L79

Comment (bramkragten) [reply src/components/ha-dialog.ts:81]: Not sure what you mean by that?

Comment (bramkragten) [reply src/components/ha-dialog.ts:81]: Is this only an issue for this specific dialog, or for every dialog?

Comment (bramkragten) [reply src/components/ha-dialog.ts:81]: That dialog doesn't use the `haStyleDialog` styles, that is the issue. You should add those styles to `static get styles()`

Comment (spacegaier) [reply src/components/ha-dialog.ts]: Thanks for pointing that out!

**Review by bramkragten — APPROVED** [review]
